package.json

```diff
@@ -8,7 +8,8 @@
     "lint": "eslint src test",
     "test": "npm run lint && mocha test/**/*-test.js",
     "watch": "npm test -- -w",
-    "lint-fix": "eslint src test --fix"
+    "lint-fix": "eslint src test --fix",
```

Comment:
Nice idea, but it's better to re-use the existing task, just like `test` and `watch`.